app/main.py

```diff
@@ -1,6 +1,11 @@
-def main():
-    # write your code here
-    pass
+def main() -> None:
+
+    file_name = input("Enter name of the file: ")
+    with open(file_name + ".txt", "w") as f:
```

Comment:
Prefer avoiding single-letter variable names, being more descriptive is a better choice.